openassessment/xblock/static/js/src/lms/oa_course_items_listing.js

```diff
@@ -17,6 +17,11 @@ export class CourseItemsListingView {
       $section.find('.open-response-assessment-msg').show();
     }
 
+    const context = this.data.CONTEXT || {};
+
+    const esgEnabled = context.ENHANCED_STAFF_GRADER;
+    const esgUrl = context.ORA_GRADING_MICROFRONTEND_URL;
```

Comment:
Naming nit: this should probably be base/root URL. URL by itself suggests to me a complete path.

```suggestion
    const esgRootUrl = context.ORA_GRADING_MICROFRONTEND_URL;
```